wheatley/row_generation/row_generator.py

```diff
@@ -65,6 +65,12 @@ def rounds(self) -> Row:
     def _gen_row(self, previous_row: Row, stroke: Stroke, index: int) -> Row:
         pass
 
+    @abstractmethod
+    def summary_string(self) -> str:
+        """ Returns a short string summarising the RowGenerator.
+        This should make grammatical sense when formatted with 'Wheatley (will|is) ringing {summary_string}.'
```

Comment:
```suggestion
        """ Returns a short string summarising the RowGenerator.
        This should make grammatical sense when formatted with 'Wheatley (will ring|is ringing) {summary_string}.'
```
Are you just logging `will ring` at the moment?
It seems a shame to have this duplicated everywhere, could a shorter version be put on the overloads?

Reply:
Yes I noticed this but at the time I couldn't really face the task of updating the comments, and it then fell by the wayside.  What I think I'll do is to leave just the first line (`Returns a short string summarising the RowGenerator.`) as the docstring for the child classes.